packages/web-awesome/src/components/TestResult/TrPwTraces/styles.scss

```diff
@@ -0,0 +1,21 @@
+.tr-metadata {
+  padding: 12px 24px 0;
+}
+
+.tr-metadata-wrapper {
+  border-bottom: 1px solid var(--on-border-primary);
+  padding-top: 8px;
+  padding-bottom: 12px;
+}
+
+.pw-trace {
+  overflow: hidden;
+  height: calc(80vh - 60px);
```

Comment:
`60px` and `75px` are magic numbers. Create variables for them to make everything obvious